components/create_comment/create_comment.jsx

```diff
@@ -549,6 +563,20 @@ export default class CreateComment extends React.PureComponent {
         this.draftsForPost[this.props.rootId] = updatedDraft;
     }
 
+    handleMouseUp = (e) => {
+        const caretPosition = Utils.getCaretPosition(e.target);
+        this.setState({
+            caretPosition,
+        });
+    }
+
+    handleKeyUp = (e) => {
+        const caretPosition = Utils.getCaretPosition(e.target);
+        this.setState({
+            caretPosition,
+        });
+    }
+
```

Comment:
One minor nit here: We can change this to be one func as `handleKeyEvent` both are functionally same. i am 0/5 on the function name here.

Reply:
@sudheerDev condensed the two functions into one. Thanks for the review :)